src/controllers/users.js

```diff
@@ -62,5 +64,13 @@ export async function login(req, res) {
       error: `The provided password is incorrect`,
     });
   }
-  return res.status(200).json({ message: `Trying to login for ${email}` });
+  var payload = {
+    "email": email,
+    "password": password
```

Comment:
Adding password to payload is an absolute threat to security. As payload can be easily viewed using sites like [jwt.io](https://jwt.io/)

Reply:
Fixed! Thank you for bringing that to my attention